ui/admin/tests/acceptance/host-catalogs/hosts/list-test.js

```diff
@@ -69,33 +66,41 @@ module('Acceptance | host-catalogs | hosts | list', function (hooks) {
     urls.newHost = `${urls.hosts}/new`;
     await authenticateSession({});
   });
+
   test('Users can navigate to hosts with proper authorization', async function (assert) {
     await visit(urls.hostCatalog);
+
     assert.ok(
       instances.hostCatalog.authorized_collection_actions.hosts.includes(
         'list',
       ),
     );
-    assert.dom(MANAGE_DROPDOWN_SELECTOR);
-    assert.ok(find(`[href="${urls.hosts}"]`));
+    assert.dom(selectors.MANAGE_DROPDOWN_HOST_CATALOG).exists();
+    assert.dom(commonSelectors.HREF(urls.hosts)).exists();
   });
 
   test('User cannot navigate to index without either list or create actions', async function (assert) {
     instances.hostCatalog.authorized_collection_actions.hosts = [];
     await visit(urls.hostCatalog);
+
     assert.notOk(
       instances.hostCatalog.authorized_collection_actions.hosts.includes(
         'list',
       ),
     );
-    assert.notOk(find(`[href="${urls.hosts}"]`));
+    assert.dom(commonSelectors.HREF(urls.hosts)).doesNotExist();
   });
 
   test('User can navigate to index with only create action', async function (assert) {
     instances.hostCatalog.authorized_collection_actions.hosts = ['create'];
     await visit(urls.hostCatalog);
-    assert.ok(find(`[href="${urls.hosts}"]`));
-    await click(MANAGE_DROPDOWN_SELECTOR);
-    assert.dom(NEW_HOST_SELECTOR).hasAttribute('href', urls.newHost);
+
```

Comment:
Enhanced this test.
We were not validating the user can actually click and navigate to the desired url, we were checking the HREF element was present with the correct url. But the test is about the user can navigate, so add that part.
Let me know yout thoughts.

Reply:
I like the enhancement of checking the link on line 98. I'm wondering if the checks on lines 100 and 104 should be moved to [this test case instead](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/host-catalogs/hosts/create-test.js#L101) in create-test.js because that test case is specific to the being able to create a new host. Thoughts?

Reply:
I would agree with what @lisbet-alvarez suggested and I am with this the test change.

Reply:
@cameronperera @lisbet-alvarez 
You guys got me thinking here. Now I do think this test shouldn't exists within this file, let me explain:

- As Lis mentioned, we already have a specific test file for creation where this case is covered. I will move the enhancement assertions to that create-test file.
- This test file is for listing, therefore, navigate to creation its out of the context of listing, as the way I understand it.

What do you guys think?

Reply:
I think that the test case is still valid because it should be making sure that a user can navigate to the `hosts` list route with only the `create` action since a user can navigate to list route with either or both. While the other test case in the `create-test.js` file was making sure that when the `create` authorization is present a user can navigate to the new host route.

Reply:
@lisbet-alvarez 
You are right, added back the test.